Add AdSpeed ad server to amp-ad

Adspeed opened the pull request:
Hello,

I've followed the guidelines and examples to add support for AdSpeed ad server into amp-ad. 

- No error with gulp lint
- Ran & passed tests "test-ads-config.js,test-integration.js"
- Tested locally with "examples/ads.amp.max.html?type=adspeed"
- Signed Google CLA (git username: "adspeed")

Please review and let me know. Thanks!

jridgewell commented:
/to @lannka

lannka commented on ads/_config.js:
```diff
@@ -84,6 +84,10 @@ export const adConfig = {
 
+  adspeed: {
+    preconnect: 'https://g.adspeed.net',
+  },
```
could you also implement the render-start and no-content API?

Adspeed commented:
Hello, I've just updated our backend JS code to call the API renderStart() and noContentAvailable() when suitable and I modified ads/_config.js. However, Travis CI build failed (error below), which I don't think is because of our code, looks like a config thing. Please check and restart build if it's the case. Thanks

"The command "node build-system/pr-check.js" exited with 1."

Adspeed commented:
Hello. Can someone help restart Travis build? Thanks!

jridgewell commented:
Restarted.

Adspeed commented:
It failed very early again at: "pr-check.js - any update to package.json or yarn.lock must include the other file. Please update through yarn."

Anyone knows why? 

I re-ran these local checks on my machine without any issue:
- gulp lint
- gulp check-types
- gulp build
- gulp serve

Thanks for your help

jridgewell commented:
/cc @erwinmombay

Adspeed commented:
Hello,

Any help with the build config/troubleshooting/result? 

Or should I close this PR and restart using the latest amphtml revision?

Thanks!

jridgewell commented:
Try rebasing against current master? Assuming you called [`ampproject/amphtml`](github.com/ampproject/amphtml) the `upstream` remote:

```bash
$ git pull --rebase upstream/master
```

Adspeed commented:
Thanks Justin! That (updating/syncing the fork) definitely helped and all checks have passed now. Please review.

Adspeed commented:
Thanks!!
